plugins/obs-transitions/transition-stinger.c

```diff
@@ -107,6 +111,8 @@ static void stinger_update(void *data, obs_data_t *settings)
 	s->do_texrender = s->track_matte_enabled && s->matte_layout < MATTE_LAYOUT_SEPARATE_FILE;
 
 	if (s->matte_source) {
+		if (s->transitioning)
+			obs_source_remove_active_child(s->source, s->matte_source);
```

Comment:
Prefer an empty line after an unenclosed `if`, or else enclose the if with curly braces.